src/dex/fluid-dex/fluid-dex.ts

```diff
@@ -19,23 +19,22 @@ import {
   DebtReserves,
   FluidDexData,
   FluidDexPool,
-  FluidDexPoolState,
+  // FluidDexPoolState,
   Pool,
 } from './types';
 import { SimpleExchange } from '../simple-exchange';
 import FluidDexPoolABI from '../../abi/fluid-dex/fluid-dex.abi.json';
 import { FluidDexConfig, FLUID_DEX_GAS_COST } from './config';
-import { FluidDexEventPool } from './fluid-dex-pool';
 import { FluidDexFactory } from './fluid-dex-factory';
 import { getDexKeysWithNetwork, getBigIntPow } from '../../utils';
 import { extractReturnAmountPosition } from '../../executor/utils';
 import { MultiResult } from '../../lib/multi-wrapper';
 import { generalDecoder } from '../../lib/decoders';
 import { BigNumber } from 'ethers';
 import { sqrt } from './utils';
+import { FluidDexLiquidityProxy } from './fluid-dex-liquidity-proxy';
 
 export class FluidDex extends SimpleExchange implements IDex<FluidDexData> {
```

Comment:
btw, should we support `deposit/withdraw` of fUSDC token in this dex integration? (like in [this transaction)](0x844c04c38ad3b8853238b735ee0eaf6b96df04dbe8911153b0d81eeb8e9a5125))

Reply:
I think no. I can see that LogOperate event was triggered within this transaction